docs/static/releasenotes.asciidoc

```diff
@@ -41,6 +42,44 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-10-0]]
+=== Logstash 8.10.0 Release Notes
+
+[[notable-8.10.0]]
+==== Notable issues fixed
+
+* Fixed bugs in DLQ 
+** throwing `java.nio.file.NoSuchFileException` when finalize the segment https://github.com/elastic/logstash/pull/15233[15233]
+** sorting DLQ file in wrong order https://github.com/elastic/logstash/pull/15246[#15246]
+
+[[dependencies-8.10.0]]
+==== Updates to dependencies
+
+* Updated JRuby to 9.4.2.0 https://github.com/elastic/logstash/pull/15283[#15283]
+* Removed custom bundler and used JRuby bundled bundler https://github.com/elastic/logstash/pull/15066[#15066]
+
+==== Plugins
```

Comment:
Something weird happened when release notes for plugins were generated.  Maybe because we're going from `x.9` to `x.10`? 

Traditional order for plugin notes is codecs, filters, inputs, integrations, outputs.  Although now that I think about it, I'd like to move integrations to the top of the list.

Reply:
all are my bad. I deleted all plugin placeholders and moved them in wrong order.